blocks/table/table.css

```diff
@@ -302,4 +302,20 @@ main .table-container .table-wrapper .table.no-border.block {
 
 .biencommun .report.table-container table td:nth-child(3) a svg {
   fill: var(--site-color-bien) !important;
+}
+
+
+/* Fondations color overrides for report table */
+
+.fondations .report.table-container table td:nth-child(2) p,
+.fondations .report.table-container table th:nth-child(2) p {
+  color: var(--fondations-site-color) !important;
+}
+
+.fondations .report.table-container table td:nth-child(3) a {
+  color: var(--site-color-bien) !important;
+}
+
+.fondations .report.table-container table td:nth-child(3) a svg {
+  fill: var(--site-color-bien) !important;
```

Comment:
please remove `!important`
As shared earlier as well, it should not be used as per best practices in Edge Delivery